cumulusci/tasks/salesforce/Deploy.py

```diff
@@ -60,23 +68,31 @@ def _include_file(self, root_parts, f):
             return lower_f.endswith((".js", ".js-meta.xml", ".html", ".css", ".svg"))
         return True
 
-    def _get_files_to_package(self, path):
+    def _get_files_to_package(self):
         for root, dirs, files in os.walk("."):
             root_parts = root.split(os.sep)[1:]
             if self._include_directory(root_parts):
```

Comment:
Should `_include_directory()` filter out the `static_resources_path` here? Or is there a different mechanism for excluding the un-archived static resources from the final .zip?

Reply:
The `static_resources_path` is meant to be a separate directory outside of the `path` being deployed.

We should add a check for that but my immediate goal is unblocking the project that needs this feature, so I'm inclined to follow up and add this in a separate PR soon.